helmfile.d/helmfile-60.teams.yaml

```diff
@@ -11,6 +11,7 @@ bases:
 
 releases:
   {{- range $teamId, $team := $tc.teams }}
+  {{- if $v.otomi.isMultitenant }}        
```

Comment:
are you sure you know what you are doing here? Why iterate over something that might be false for all iterations? Might be better to put it one line higher.

But on another note: This effectively disables all of the functionality for teams if `isMultitenant`, and on which assumption did you base that decision? Please consult for full understanding in case you don't have clarity on the specs.

Reply:
When `isMultitenant=false` the concept of 'teams' still exist, as it is merely a way of grouping (team/project/whatever) resources, but we don't need every team to get their own telemetry and other heavy stuff.

Reply:
Err ok, I'll just move it back to where it came from and put team jobs outside of that scope.